### DISCUSSION TITLE
where could i find examples about using dakota as third party library

### DISCUSSION THREAD

**Discussion by supermenxp** [snl-dakota/dakota#60] [Q&A] [answered]
What should I do to integrate Datota into my application

**Comment by jadamstephens** [comment, marked as answer]
Hi,

The file [library_mode.cpp](https://github.com/snl-dakota/dakota/blob/devel/src/library_mode.cpp) demonstrates a few different cases.

**Reply by supermenxp** [reply]
thanks
